packages/bpk-component-accordion/src/__snapshots__/BpkAccordionItem-test.js.snap

```diff
@@ -34,12 +34,13 @@ exports[`BpkAccordionItem should not render an "initiallyExpanded" attribute on
             >
               <svg
                 aria-hidden="true"
-                style="width: 1rem; height: 1rem;"
+                height="1rem"
                 viewBox="0 0 24 24"
+                width="1rem"
                 xmlns="http://www.w3.org/2000/svg"
               >
                 <path
-                  d="M19.113 8.095a1.496 1.496 0 010 2.008l-6.397 5.948a1 1 0 01-1.358.003l-6.532-6.01a1.427 1.427 0 01.138-1.949 1.572 1.572 0 011.997-.103l5.078 4.638 4.97-4.535a1.72 1.72 0 012.104 0z"
+                  d="M19.113 8.095a1.496 1.496 0 0 1 0 2.008l-6.397 5.948a1 1 0 0 1-1.358.003l-6.532-6.01a1.427 1.427 0 0 1 .138-1.949 1.57 1.57 0 0 1 1.997-.103l5.078 4.638 4.97-4.535a1.72 1.72 0 0 1 2.104 0"
```

Comment:
Is a path change like this to be a concern?

Reply:
No concern, the diff is within an `a` command: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d#path_commands for drawing arcs. The previous parser/generator concatenated the `large-arc-flag` and `sweep-flag` (either `0` or `1`) and the new parser/generator does not. Both are valid and do not change the resulting svg viewed. 

In addition I can see that some path values (`1.572`) have been shortened (`1.57`). My guess is that this is an optimisation step performed during generation & accuracy to 3 decimal places does not increase visual clarity so it is optimised away.